src/components/screens/NewsItem.js

```diff
@@ -59,7 +59,7 @@ export const NewsItem = ({ data, navigation }) => {
   );
 
   const businessAccount = dataProvider?.dataType === 'business_account';
-
+  const a11yText = consts.a11yLabel;
```

Comment:
as a general rule of thumb, we have an empty line after blocks of consts in our code, so please add one here again

Reply:
solved f90ab7f

Reply:
and here

Reply:
now solved f90ab7f